test/integ/quote-classic.test.ts

```diff
@@ -1838,153 +1834,140 @@ describe('quote', function () {
-                        type,
-                        slippageTolerance: SLIPPAGE,
-                        sendPortionEnabled: sendPortionEnabled,
-                        configs: [
-                          {
-                            routingType: RoutingType.CLASSIC,
-                            recipient: alice.address,
-                            deadline: 360,
-                            simulateFromAddress: '0xf584f8728b874a6a5c7a8d4d387c9aae9172d621',
-                            algorithm,
-                            enableUniversalRouter: true,
-                          },
-                        ],
-                      };
-
-                      const response = await call(quoteReq);
-                      const { data, status } = response;
-                      const quoteJSON = data.quote as ClassicQuoteDataJSON;
-
-                      expect(status).to.equal(200);
-                      expect(quoteJSON.simulationError).to.equal(false);
-                      expect(quoteJSON.methodParameters).to.not.be.undefined;
-
-                      if (sendPortionEnabled) {
-                        // portion recipient must be undefined
-                        expect(quoteJSON.portionRecipient).to.be.undefined;
-
-                        // all other fields must be defined for clients to know that portion bips is 0%.
-                        expect(quoteJSON.portionBips).to.not.be.undefined;
-                        expect(quoteJSON.portionAmount).to.not.be.undefined;
-                        expect(quoteJSON.portionAmountDecimals).to.not.be.undefined;
-                        expect(quoteJSON.quoteGasAndPortionAdjusted).to.not.be.undefined;
-                        expect(quoteJSON.quoteGasAndPortionAdjustedDecimals).to.not.be.undefined;
+                it(`stable-to-stable ${tokenIn.symbol} -> ${tokenOut.symbol} sendPortionEnabled = ${sendPortionEnabled}`, async () => {
+                  const originalAmount = '10';
+                  const tokenInSymbol = tokenIn.symbol!;
+                  const tokenOutSymbol = tokenOut.symbol!;
+                  const tokenInAddress = tokenIn.isNative ? tokenInSymbol : tokenIn.address;
+                  const tokenOutAddress = tokenOut.isNative ? tokenOutSymbol : tokenOut.address;
+                  const amount = await getAmountFromToken(type, tokenIn.wrapped, tokenOut.wrapped, originalAmount);
+                  const getPortionResponse = await portionFetcher.getPortion(
+                    tokenIn.chainId,
+                    tokenInAddress,
+                    tokenOut.chainId,
+                    tokenOutAddress
+                  );
+
+                  // TODO: remove the if statement, once portion service doesn't return portion for stable-to-stable
+                  if (sendPortionEnabled) {
+                    expect(getPortionResponse.hasPortion).to.be.false;
+                    expect(getPortionResponse.portion).to.be.undefined;
+                  }
```

Comment:
seems like we should remove this case now that we fixed the stable stable carveout in portion service?

Reply:
great catch Eric, agreed